libraries/persistent_store/src/lib.rs

```diff
@@ -12,299 +12,297 @@
+//!     -   A [bit](format::HEADER_DELETED) indicating whether the entry is deleted.
+//!     -   A [bit](format::HEADER_FLIPPED) indicating whether the value is word-aligned and has all
+//!         bits set to 1 in its last word. The last word of an entry is used to detect that an
+//!         entry has been fully written. As such it must contain at least one bit equal to zero.
+//!     -   The [key](format::HEADER_KEY) of the entry.
+//!     -   The [length](format::HEADER_LENGTH) in bytes of the value. The value follows the header.
+//!         The entry is word-aligned if the value is not.
+//!     -   The [checksum](format::HEADER_CHECKSUM) of the first and last word of the entry.
+//! -   [Erase](format::ID_ERASE): A word used during compaction. It contains the
+//!     [page](format::ERASE_PAGE) to be erased and a [checksum](format::WORD_CHECKSUM).
+//! -   [Clear](format::ID_CLEAR): A word used during the clear operation. It contains the
+//!     [threshold](format::CLEAR_MIN_KEY) and a [checksum](format::WORD_CHECKSUM).
+//! -   [Marker](format::ID_MARKER): A word used during a transaction. It contains the [number of
+//!     updates](format::MARKER_COUNT) following the marker and a [checksum](format::WORD_CHECKSUM).
+//! -   [Remove](format::ID_REMOVE): A word used inside a transaction. It contains the
+//!     [key](format::REMOVE_KEY) of the entry to be removed and a
+//!     [checksum](format::WORD_CHECKSUM).
 //!
 //! Checksums are the number of bits equal to 0.
 //!
 //! # Proofs
 //!
 //! ## Compaction
 //!
-//! It should always be possible to fully compact the store, after what the
-//! remaining capacity should be available in the current window (restoring the
-//! compaction invariant). We consider all notations on the virtual storage after
-//! the full compaction. We will use the `|x|` notation although we update the state
-//! of the virtual storage. This is fine because compaction doesn't change the
-//! status of an existing word.
+//! It should always be possible to fully compact the store, after what the remaining capacity
+//! should be available in the current window (restoring the compaction invariant). We consider all
+//! notations on the virtual storage after the full compaction. We will use the |x| notation
+//! although we update the state of the virtual storage. This is fine because compaction doesn't
+//! change the status of an existing word.
 //!
-//! We want to show that the next `N - 1` compactions won't move the tail past the
-//! last page of their window, with `I` the initial window:
+//! We want to show that the next N - 1 compactions won't move the tail past the last page of their
+//! window, with I the initial window:
 //!
-//! ```text
-//! forall 1 <= i <= N - 1, t_{I + i} <= (I + i + N - 1) * Q
-//! ```
+//! |                  |            |   |                     |
+//! | ----------------:| ----------:|:-:|:------------------- |
+//! | ∀(1 ≤ i ≤ N - 1) | t\_{I + i} | ≤ | (I + i + N - 1) × Q |
```

Comment:
These vertical lines are not aligned. I wonder if we want that.

Reply:
Weird, for me they are. But I know I didn't do it at other places where it would make the lines very wide.

Reply:
![alignment](https://user-images.githubusercontent.com/43844792/111149431-286c4800-858d-11eb-9056-ffbac3789cf0.png)

This is what I see.

Reply:
Oh I see, this is because you don't have a monospace font that supports UTF-8.

Reply:
Oh true, the `∀` is actually pretty wide.